src/controllers/payments.js

```diff
@@ -140,9 +141,11 @@ exports.put = function *(id) {
 
   payment.id = id.toLowerCase()
 
-  let isPrepared = yield Payments.validate(payment)
-  if (isPrepared) {
-    yield Payments.settle(payment)
+  yield Payments.validate(payment)
+  const truePayment = settlementQueue.storePayment(payment)
```

Comment:
Sounds better, how does it become trusted? Once it goes in the queue it becomes trusted?